packages/components/src/Tag/Tag.tsx

```diff
@@ -44,14 +48,15 @@ type Props = {
 /**
  * This component provides a tag (pill shaped badge) wrapper.
  */
-function Tag({ color, children, icon, variant = "flat" }: Props) {
+function Tag({ color, children, className, icon, variant = "flat" }: Props) {
   return (
     <Text
       as="span"
       className={clsx(
         styles.tag,
         color && stylesByColor[color],
         variant === "outline" && styles.outline,
+        className,
```

Comment:
nit: could you move to line 56 so it's first in the list? (just for consistency)